TWLight/resources/templates/resources/partner_detail.html

```diff
@@ -175,14 +175,12 @@ <h1>{{ object }}</h1>
               We filter out details and summary tags so that we always see
               the full content on the details page.
             {% endcomment %}
-            {% filter twlight_removetags:"details summary" %}
```

Comment:
Is this filter no longer needed? If so, the comment above should be removed if it's no longer relevant.

Reply:
Removed comment. I reckon we won't be using the details and summary tag in the short description (eg., EBSCO)